CI/appveyor.after_build.sh

```diff
@@ -0,0 +1,187 @@
+#!/bin/sh
+
+# This assembles all the files we need into a package sub-directory
+
+echo "Running appveyor.after_build.sh shell script..."
+
+echo "  ccache statistics after end of compilation"
+${CCACHE} -s
+echo ""
+
+if [ ${APPVEYOR_REPO_NAME} != "Mudlet/Mudlet" ] ; then
+    # Only run this code on the main Mudlet Github repository - do nothing otherwise:
+    echo "This does not appear to be running on the main Mudlet repository, packaging is not appropriate....!"
+    echo ""
+    exit 0
+fi
+
+# Source/setup some variables (including PATH):
+. $(/usr/bin/cygpath --unix ${APPVEYOR_BUILD_FOLDER}/CI/appveyor.set-build-info.sh)
+
+echo "Moving to packaging directory: $(/usr/bin/cygpath --windows ${APPVEYOR_BUILD_FOLDER}/package)"
+cd $(/usr/bin/cygpath --unix ${APPVEYOR_BUILD_FOLDER}/package)
+echo "  it contains:"
+ls -l
+echo ""
+
+# Since Qt 5.14 using the --release switch is broken (it now seems to be
+# assumed), --debug still seems to work.
+# https://bugreports.qt.io/browse/QTBUG-80806 seems relevant.
+echo "Running windeployqt..."
+${MINGW_INTERNAL_BASE_DIR}/bin/windeployqt --no-virtualkeyboard mudlet.exe
+echo ""
+
+echo "Copying system libraries in..."
+if [ "${BUILD_BITNESS}" = "32" ] ; then
+    cp -v -p ${MINGW_INTERNAL_BASE_DIR}/bin/libgcc_s_dw2-1.dll .
```

Comment:
manually putting all dependencies here seems to be cumbersome... I am sure added a script to an issue, that automated the process using `ntldd`. I'd prefer that to listing all dependencies manually.

Reply:
Trouble is I tried that - and found it doesn't pick up on the dynamically loaded libraries!

These at least includes: the Open SSL ones, the Discord RPC one and anything that is needed for the Lua subsystem... 

It would be nice to script something but I am not sure it will save us much effort. As it is, if something gets updated so that a file is NOT there will be an error in the build log - perhaps I need to include a status check on the copying commands so that the build job automatically fails if any of them do...

Reply:
The dynamically linked libraries can be manually copied and ran with the same script to copies their dependencies over. That would gain us quite a bit, just look at your list.

> As it is, if something gets updated so that a file is NOT there will be an error in the build log - perhaps I need to include a status check on the copying commands so that the build job automatically fails if any of them do...

With the script, most of the time nothing needs to be changed around the CI scripts, which MUCH better in my opinion.

Reply:
Script for dependencies: https://github.com/Mudlet/Mudlet/pull/3255#issuecomment-681547576